pkg/storaged/client.js

```diff
@@ -960,21 +962,38 @@ function stratis3_start() {
     client.stratis_manager.StoppedPools = {};
 
     return client.stratis_manager.wait()
+            .catch(err => {
+                if (stratis3_interface_revision == stratis3_interface_revision_newest &&
+                !client.stratis_manager.valid) {
+                    stratis3_interface_revision = stratis3_interface_revision_fallback;
+                    return stratis3_start();
+                }
+                return Promise.reject(err);
+            })
             .then(() => {
                 client.stratis_store_passphrase = (desc, passphrase) => {
                     return python.spawn(stratis3_set_key_py, [desc], { superuser: true })
                             .input(passphrase);
                 };
 
                 client.stratis_start_pool = (uuid, unlock_method) => {
-                    return client.stratis_manager.StartPool(uuid, [!!unlock_method, unlock_method || ""]);
+                    if (stratis3_interface_revision >= 4)
+                        return client.stratis_manager.StartPool(uuid, "uuid", [!!unlock_method, unlock_method || ""]);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18852-20230606-092422-9718ac32-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)